interface/src/app/help/help.tpl.html

```diff
@@ -3,50 +3,48 @@
   <br>
   <div class="lead" id="Genes">
     <h3><u>Genes</u></h3>
-    Gene content goes here. Lorem ipsum dolor sit amet, consectetur adipiscing
-    elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
-    Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut
-    aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in
-    voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint
-    occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit
-    anim id est laborum.
+    ADAGE model is built with genes in Pseudomonas aeruginosa PAO1 strain.
+    Information about genes are obtained from the following NCBI website:
+    ftp://ftp.ncbi.nih.gov/gene/DATA/GENE_INFO/Archaea_Bacteria/Pseudomonas_aeruginosa_PAO1.gene_info.gz
```

Comment:
Would it be easier to say that information about genes is obtained from NCBI, where NCBI is a link to that URL?